.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!--
+  The description should provide all necessary information for a reviewer.
+  - What does this PR change, what's the reason for the change, how can it be tested
+-->
+&lt;Add detailed description for reviewers here.&gt;
+
+
+### Dependency release notes
+
+<!-- add links to release notes if important dependencies changed -->
+N/A
+
+### Submitter checklist
+
+- [ ] Change has been tested (on a back-end cluster)
+- [ ] [changelog.yaml] with upgrade notes are prepared and appropriate for the audience affected by the change (users or developer, depending on the change).
+- [ ] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
+  - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml)
```

Comment:
```suggestion
  - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml)
      - version: 
      - appVersion: 
```

Reply:
Both?

Reply:
We always release the steward controller images and Helm chart together, right? So yes, both. I'll add it.

Reply:
```suggestion
  - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)
```